lib/Doctrine/EntityManagerFactory.php

```diff
@@ -43,10 +45,15 @@ public function __construct(
     $this->emojiEncodingListener = $emojiEncodingListener;
   }
 
-  public function createEntityManager() {
+  public function createEntityManager(): EntityManager {
     $entityManager = EntityManager::create($this->connection, $this->configuration);
     $this->setupListeners($entityManager);
-    if (class_exists(Debugger::class)) {
+    if (
+      class_exists(Debugger::class)
+      && class_exists(DoctrinePanel::class)
+      && class_exists(DIPanel::class)
```

Comment:
I'm not sure if we need to check `DIPanel` and `ApiPanel` existence here because they're not referenced in the file or inside `DoctrinePanel`. But there are other files where these panels are explicitly instantiated, they will likely also break so we need safety checks in them too:
 https://github.com/mailpoet/mailpoet/blob/master/lib/API/JSON/API.php#L194-L197
 https://github.com/mailpoet/mailpoet/blob/master/lib/AdminPages/PageRenderer.php#L67-L69